nat-lab/tests/mesh_api.py

```diff
@@ -273,9 +275,14 @@ def config_dynamic_nodes(self, node_configs: List[bool]) -> Tuple[Node, ...]:
                 public_key=str(public),
                 is_local=is_local,
             )
-            # TODO correct subnet when we'll decide about the range
-            self.assign_ip(node.id, f"100.64.33.{node_idx}")
-            self.assign_ip(node.id, f"fd00::dead:{node_idx}")
+            ipv4 = (
+                f"100.{random.randint(65, 127)}.{random.randint(0, 255)}.{random.randint(8, 255)}"
```

Comment:
Not including `64` in second octet, and last octet should exclude `255` which is broadcast, but these are nitts/edge cases.

Reply:
updated